boxoffice/views/order.py

```diff
@@ -314,6 +315,35 @@ def format_assignee(line_item):
     return api_orders
 
 
+# TODO activate route when front-end becomes available
+# @app.route('/line_item/<line_item_id>/cancel', methods=['POST'])
+# @load_models(
+#     (Order, {'id': 'line_item_id'}, 'line_item')
+#     )
+def cancel_line_item(line_item):
+    if not line_item.is_confirmed:
```

Comment:
Shouldn't we also disallow line items from being cancelled, after an event has completed (perhaps with a grace of one day)? If so, it makes sense to add `start_at` and `end_at` to an `ItemCollection`?